numismatic/requesters.py

```diff
@@ -21,6 +24,20 @@
 class Requester:
     "Basic Requester using requests and blocking calls"
 
+    retries = attr.ib(default=attr.Factory(
+        config_item_getter('REQUESTER', 'retries')),
```

Comment:
Nice! For the other classes I tried to have the ini file section have the same title as the class name (i.e. the same case). Not sure if that runs foul of INI file conventions of being all upper case. The aim was that with the ConfigMixin it would automatically find the correct section and be open for extension by subclasses. It was a bit tricky and I see that for the api keys in Luno and BraveNewCoin we didn't follow that. However it does seem to work for assets and currencies (e.g. for Luno) but I think you have to define a custom validator.

Reply:
Donald Trump's famous quote (and what I tell my team often) may apply here: **Premature optimization is the root of all evil**. My take is that we may be overthinking things a bit here. Lets just try make things simple for now. In that regard, I propose making things all capitals (like the ConfigSetter examples online) and have the spelling identical to the class name.

Also, those custom validators look like code bloat to me...